### DISCUSSION TITLE
GitHub Wikis' default branch name is still `master`

### DISCUSSION THREAD

**Discussion by ero-qt** [community/community#128945] [Repositories]
### Select Topic Area

Bug

### Body

GitHub has switched their default branch naming from `master` to `main` a long time ago.  
However, the same cannot be said of a repository's wiki. When cloning `https://github.com/some-user/some-repo.wiki.git`, we can see that the default branch is still named `master`. Since there is no way to switch branches for wikis, this blemish will always remain locally.

Please change this so as to be consistent with the rest of the website.

**Comment by peeter7** [comment]
You're right, GitHub wikis currently use "master" as the default branch name, which isn't consistent with the main branch standard. While switching branches isn't possible for wikis, there are ways to achieve consistency:

**1. Accept the minor inconsistency:** Branch names in wikis are purely for local reference and don't affect functionality. You can choose to accept this minor inconsistency for the sake of simplicity.
**2. Update the local branch name (informational):** While not functionally changing anything, you can rename the local "master" branch to "main" for informational purposes. This keeps things consistent with your local workflow.

Here's how:

- Navigate to your cloned wiki directory.
- Use the following command: git branch -m master main
- This renames the local "master" branch to "main".

**Important points to remember:**

1. Renaming the local branch doesn't affect the remote wiki on GitHub.
2. This is purely for your own reference and doesn't change how the wiki functions.

While GitHub may eventually update wiki branch [names](http://universalnamegenerator.com/), there's no official timeline yet.  For now, choosing between accepting the minor inconsistency or renaming the local branch is up to your preference.
